Firestore/core/src/firebase/firestore/nanopb/reader.h

```diff
@@ -189,6 +258,17 @@ T Reader::ReadNestedMessage(const std::function<T(Reader*)>& read_message_fn) {
   return message;
 }
 
+template <typename T, typename C>
+T Reader::ReadNestedMessageImpl(
+    const C& serializer,
+    Reader::ReadingMemberFunction<T, C> read_message_member_fn) {
+  std::function<T(Reader*)> read_message_fn = [=](Reader* reader) {
```

Comment:
`serializer` will be copied, is that intentional?

Reply:
no; done.